ci/Jenkinsfile

```diff
@@ -165,13 +165,13 @@ def testAndBuildFrontend() {
                     stash includes: 'frontend/dist/**/*', name: 'frontend'
                 }
 
-                stage('typedoc') {
-                    sh 'cd frontend && npm run doc'
+                stage('build-typedoc') {
+                    sh 'make generate-type-doc'
                     stash includes: 'frontend/src/generated/docs/', name: 'typedoc'
                 }
```

Comment:
I don't think this stage or artifact is needed. Afaik `npm run doc` is only needed before storybook to build storybook correctly.